### DISCUSSION TITLE
Welcome to spark Discussions!

### DISCUSSION THREAD

**Discussion by christoph-v** [christoph-v/spark#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other people. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other people.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by coderextreme** [comment]
I’m coderextreme.   I do JavaScript and probably TypeScript.   I’m trying to port a peer-to-peer video app from Angular 2 alpha to Angular 13.   It’s called “Grasshopper Academy.”  So far, there are still bugs in the build.

**Reply by coderextreme** [reply]
If anyone wants to help debugging I can check in the code.  Let me know!

**Reply by christoph-v** [reply]
Could you create a new Discussions topic, where you can introduce the Grasshopper Academy to us?
In particular I would like to know about the differences between Node.js and Angular.

**Comment by christoph-v** [comment]
I am Christoph V.
I have stored a lot of my ideas (spark, psark, digits) in the repositories and am now hoping to find some time to make "real" software from those ideas. A predecessor of these ideas is the project SrrTrains v0.01, which was hosted at another platform (not GitHub):
[https://letztersein.com/srrtrains-v0-01](https://letztersein.com/srrtrains-v0-01)
